spec/abstract_controller_base_spec.rb

```diff
@@ -74,9 +75,28 @@
           CREATE_CALL
         end
 
+        it 'records and labels view rendering' do
+          expect(File).to exist(list_users_appmap_json)
+          appmap = JSON.parse(File.read(list_users_appmap_json)).to_yaml
+
+          expect(appmap).to match(<<-VIEW_CALL.strip)
+  event: call
+  thread_id: .*
+  defined_class: ActionView::Renderer
+  method_id: render
+  path: .*
+  lineno: .*
+  static: false
+          VIEW_CALL
+
+          expect(appmap).to match(<<-VIEW_LABEL.strip)
+          "labels":["view"]
+          VIEW_LABEL
+        end
```

Comment:
Can squiggly heredocs be used here so it's indented nicely?

Reply:
The indentation matters when matching YAML as a string. Probably a nicer way to do it, but ...

Reply:
But these are unindented at the base level, no? And squiggly heredocs remove the extra indentation.